### PR TITLE
Composio: add new example

### PR DESCRIPTION
**Pull request by Karthikeya-Meesala**
## Why are these changes needed?
Our blog post shows how Autogen Agents can automate tasks with Linear and GitHub using Composio.dev. We believe this workflow could help others, adding it to Autogen's gallery would inspire the community.

### CODE REVIEW

**Comment by Karthikeya-Meesala** [comment]
> @Karthikeya-Meesala please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree company="Composio"

File: website/src/data/gallery.json [hunk 1]
```diff
@@ -198,5 +198,12 @@
     "description": "A multimodal webbrowsing agent that autonomously browses the web.",
     "image": "webagent.jpg",
     "tags": ["tools", "app"]
+  },
+  {
+    "title": "Create Issues from Code Commits - using Autogen",
+    "link": "https://blog.composio.dev/automating-task-creation-with-openai-assistant-from-code-comments-to-linear-issues/",
+    "description": "Discover how to automatically create Linear issues from GitHub commits using Autogen Agents.",
```
Comment (ekzhu) [inline website/src/data/gallery.json:205]:
```suggestion
    "description": "Automatically creating Linear issues from GitHub commits using AutoGen Agents.",
```

Comment (ekzhu) [reply website/src/data/gallery.json:205]: Can you expand a bit more on "Linear issues" in the description.

Comment (Karthikeya-Meesala) [reply website/src/data/gallery.json]: Hey Eric, Just updated in this [commit](https://github.com/microsoft/autogen/pull/2237/commits/9f48789a51790957431b6900705c761986c26c8e). Please review it.

**Review by ekzhu — APPROVED** [review]

**Pull request merged**